lib/api-list-data.js

```diff
@@ -212,6 +221,10 @@ module.exports = class ApiListData extends API {
 		}
 	}
 
+	shouldCallFormatter(dataWithDefaults) {
```

Comment:
este método lo hiciste así como para poder escalarlo y en algún momento redefinirlo o algo el param, te pregunto porque sino es ir derecho por `this.dataWithDefaults` en la 225

Reply:
Si, si alguien el día de mañana tuviera que definir si llamar o no en base a que campos se filtraron, queda mas claro recibir la data como parametro y no usar un `this.dataWithDefaults` medio fantasma y que no es parte de la API del paquete..